### PR TITLE
Add lbstm support for create-ns

### PR DESCRIPTION
**Pull request by bjpaupor**
Signed-off-by: Brandon Paupore <brandon.paupore@wdc.com>

### CODE REVIEW

File: src/nvme/types.h [hunk 1]
```diff
@@ -1743,7 +1743,8 @@ struct nvme_id_ns {
 	__u8			nguid[16];
 	__u8			eui64[8];
 	struct nvme_lbaf	lbaf[64];
-	__u8			vs[3712];
+	__le64			lbstm;
```
Comment (igaw) [inline src/nvme/types.h:1746]: Please also update the documentation accordingly.

File: src/nvme/util.c [hunk 2]
```diff
@@ -402,7 +402,7 @@ void nvme_init_dsm_range(struct nvme_dsm_range *dsm, __u32 *ctx_attrs,
 }
 
 void nvme_init_id_ns(struct nvme_id_ns *ns, __u64 nsze, __u64 ncap, __u8 flbas,
-		__u8 dps, __u8 nmic, __u32 anagrpid, __u16 nvmsetid)
+		__u8 dps, __u8 nmic, __u32 anagrpid, __u16 nvmsetid, __u64 lbstm)
```
Comment (igaw) [inline src/nvme/util.c:405]: As this is breaking the API, do we really need to add lbstm?  Actually I wonder if we shouldn't just remove the
function as nvme_id_ns has many members and this function is likely always missing out for someone. The only user for this function I found is in create_ns() in nvme-cli. We could easily ditch this function IMO.

**Review by igaw — CHANGES_REQUESTED** [review]

File: src/nvme/util.c (hunk 2 above)
Comment (bjpaupor) [reply src/nvme/util.c]: Taking it out and handling this in nvme-cli makes sense to me, removed the function.

File: src/nvme/types.h (hunk 1 above)
Comment (bjpaupor) [reply src/nvme/types.h:1747]: Thanks for the catch, updated.

File: src/nvme/util.c (hunk 2 above)
Comment (igaw) [reply src/nvme/util.c]: This should be a new commit. It has nothing to do with introducing the lbstm into struct nvme_id_ns. 
Also can nvme_init_id_ns also remove from the src/libnvme.map export file? Obviously nvme-cli needs a fix as well.

Thanks!

**Comment by igaw** [comment]
Looks good. Let's get the nvme-cli change aligned and then merge both changes at the same time. Thanks!

**Pull request merged**